mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/dsl/syntax/DslSyntaxUtils.java

```diff
@@ -40,6 +43,10 @@
 
   private DslSyntaxUtils() {}
```

Comment:
do we really need this utils to be in the API? We should try our best to remove it from here. This is purely implementation and any bug here would force releasing a new API version, whereas the API hasn't actually changed, only an internal utils